recipe/common.php

```diff
@@ -69,12 +69,19 @@
 
 
 /**
- * Return current release path.
+ * Custom paths.
  */
-set('current_path', function () {
-    $link = run("readlink {{deploy_path}}/current")->toString();
-    return substr($link, 0, 1) === '/' ? $link : get('deploy_path') . '/' . $link;
-});
+set('current_dir', 'current');
+set('release_dir', 'release');
+set('releases_dir', 'releases');
+set('shared_dir', 'shared');
+set('dep_dir', '.dep');
```

Comment:
The reason to create variables for all directories is to be consistent in the code using those directories, like using a variable to avoid repetition of a constant value.